scintillometry/dispersion/tests/test_dispersion.py

```diff
@@ -24,14 +24,22 @@ def test_quantity(self):
         assert dm_differentunit == dm
         with pytest.raises(u.UnitsError) as excinfo:
             DispersionMeasure(self.dm_val * u.s)
-        assert "dispersion measures should have units" in str(excinfo.value)
+        assert "require units equivalent to 'pc / cm3'" in str(excinfo.value)
```

Comment:
And this is the risk of checking exception messages....

Reply:
Yes, but better to check regardless?

Reply:
Sure!